enterprise/api/v1/serializers.py

```diff
@@ -97,3 +97,25 @@ class Meta:
     user = UserSerializer()
     enterprise_customer = EnterpriseCustomerSerializer()
     data_sharing_consent = UserDataSharingConsentAuditSerializer(many=True)
+
+
+class EnterpriseCustomerUserEntitlementSerializer(serializers.Serializer):
+    """
+    Serializer for the entitlements of EnterpriseCustomerUser.
+
+    This Serializer is for read only endpoint of enterprise learner's entitlements
+    It will ignore any state changing requests like POST, PUT and PATCH.
+    """
+    entitlements = serializers.ListField(
+        child=serializers.IntegerField()
+    )
+
+    def create(self, validated_data):
+        """
+        Do not perform any operations for state changing requests.
+        """
+
```

Comment:
syntactically it is okay to skip `pass` if you have a doc-string in a no-op function.